x-pack/platform/plugins/shared/alerting/server/routes/gaps/apis/find/transforms/transform_response/v1.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { FindGapsResponseBodyV1 } from '../../../../../../../common/routes/gaps/apis/find';
+
+import { Gap } from '../../../../../../lib/rule_gaps/gap';
+
+export const transformResponse = ({
```

Comment:
can we get a unit test for this transform since it does more than just rename camel to snake case?